nx/include/switch/applets/swkbd.h

```diff
@@ -18,9 +18,15 @@ typedef enum {
 
 /// Type of keyboard.
 typedef enum {
-    SwkbdType_Normal = 0,  ///< Normal keyboard.
-    SwkbdType_NumPad = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
-    SwkbdType_QWERTY = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Normal    = 0,  ///< Normal keyboard.
+    SwkbdType_NumPad    = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
+    SwkbdType_QWERTY    = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Unkown    = 3,  ///< The same as SwkbdType_Normal keyboard.
```

Comment:
Unknown

Reply:
I am so sorry for making so many spelling mistakes.
Thank you for your time, man.